Frends.SFTP.ListFiles/Frends.SFTP.ListFiles/Enums/AuthenticationType.cs

```diff
@@ -1,4 +1,4 @@
-﻿namespace Frends.SFTP.ListFiles.Enums;
+﻿namespace Frends.SFTP.ListFiles.Definitions;
```

Comment:
Namespace should be Enums, since the files are in Enums-folder, not in Definitions.